New Bank/NewBank/newbank/server/NewBankClientHandler.java

```diff
@@ -31,13 +31,16 @@ public void run() {
 			out.println("Checking Details...");
 			// authenticate user and get customer ID token from bank for use in subsequent requests
 			CustomerID customer = bank.checkLogInDetails(userName, password);
+
+			out.println("Enter the customer name to whom the payment needs to be made: ");
+			String customerPayment = in.readLine();
```

Comment:
This string is being printed first thing when entering the program. Users may type a different command than PAY. "Log In Successful. What do you want to do?" should be the first string printed.

Reply:
Can include this in the input argument.